src/test/stateful_test_utils.hpp

```diff
@@ -120,6 +142,23 @@ class DummyStatefulModel {
         inferRequest.SetBlob("input", inputBlob);
         inferRequest.Infer();
     }
+
+    static void setVariableState_2(ov::runtime::InferRequest& inferRequest, std::vector<float> values) {
+        DummyStatefulModel::resetVariableState_2(inferRequest);
+        std::vector<size_t> shape{1, 1};
+
+        ov::runtime::Tensor tensor(
+            ov::element::Type_t::f32,
+            shape,
+            values.data());
+        try{
+            inferRequest.set_tensor("input", tensor);
+        }
+        catch(const ov::Exception& e){
+            std::cout << "infer_request.set_tensor failed with message: " << e.what() << std::endl;
```

Comment:
In OpenVINO, if we do not set the input before inference, it will be created automatically.
The test, before your changes, tested that set blob and inference work correctly.
With this change, `try catch` catches the exception and proceeds with inference, which alters the behavior of the test.
This only covers the issue we currently have?